packages/rocketchat-livechat/server/lib/QueueMethods.js

```diff
@@ -131,5 +131,63 @@ RocketChat.QueueMethods = {
 		RocketChat.models.Rooms.insert(room);
 
 		return room;
+	},
+	'Bot_First'(guest, message, roomInfo) {
```

Comment:
Any suggestion where that function should go? I'm new here so I don't know how you guys structure your code yet, still have some reading to go